lib/dashboards/quote-amounts-widgets.ts

```diff
@@ -0,0 +1,133 @@
+import { ChainId, ID_TO_NETWORK_NAME } from '@uniswap/smart-order-router'
+import _ from 'lodash'
+import { Widget } from './core/model/widget'
+import { WidgetsFactory } from './core/widgets-factory'
+
+enum TradeTypes {
+  ExactIn = 'ExactIn',
+  ExactOut = 'ExactOut',
+}
```

Comment:
There should be a named export for these in `@uniswap/sdk-core`
```
import { TradeType } from '@uniswap/sdk-core';
```
And you can do `TradeType.EXACT_INPUT`, etc.

Reply:
I need to map them to their String value (as that's how the metric is named), but I agree that some consolidation is necessary here. Do you mind if we keep it as is for now and consolidate as a follow up?